downloader/src/test/kotlin/vcs/CvsTest.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.downloader.vcs
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class CvsTest : StringSpec({
```

Comment:
For my taste, testing that the right instance is passed which comes at the cost of quite a bit boiler plate and exposing the commandLineTool property does not provide much value in return. In particular it seems a bit unbalanced, as this is the only thing the test checks. 

I'd prefer to drop this, leaving it up to you anyway.

Reply:
Dropped the tests and made the property private.